### PR TITLE
Merge ES6 changes into voa-progress-form; fix tests.

### PR DESCRIPTION
**Pull request by awong-dev**
Brings the voa-progress-form branch in line with the Javascript environment changes being pushed into mainline.

Also greens-up the tests.  This will make it easier to continue developing.

### CODE REVIEW

File: _health-care/_js/_form_progress.js [hunk 1]
```diff
@@ -1,73 +1,63 @@
...
 
-  var initForm = function() {
-    $('.content.active').find('.form-section').first().show();
+  $(nextSection).show();
+};
 
-    $('#continueButton, #skipButton').on('click', function() {
-      showNextSection();
-    });
+export function init() {
```
Comment (akainic) [inline _health-care/_js/_form_progress.js]: Cool! Didn't realize you could do this

Comment (awong-dev) [inline _health-care/_js/_form_progress.js]: Only in ES6!!!

It replaces the whole module pattern hack we had been using previously. See

http://blog.teamtreehouse.com/get-started-ecmascript-6

**Comment by akainic** [comment]
LGTM!

**Comment by awong-dev** [comment]
bombs away...

**Pull request merged**